src/pages/concept-list-page/concept-list-page-pure.tsx

```diff
@@ -36,9 +37,11 @@ export const ConceptListPagePure = ({ history, concepts, publisher, catalogId }:
     <div className="row mb-2">
       <ConceptTitle title={getTitle(publisher)} />
     </div>
-    <div className="mb-2">
-      <NewConceptButton parentOnClick={() => createNewConceptAndNavigate({ history, catalogId })} />
-    </div>
+    <Can I="create" a="Concept">
```

Comment:
du trenger virksomhet kontekst på et sted. Rollene er per pubisher.

Reply:
din ser mye bedre ut, jeg hadde  gjort sånn :D 
```
<RequirePermission to={{permission:"createConcept", resourceType="publisher",resourceId=orgNr}}>
```

Reply:
Pr. virksomhet, da må det bli slik: <Can I="create" of={{ publisher: catalogId, __type: 'Concept' }}>